pybind11/__main__.py

```diff
@@ -1,3 +1,5 @@
+# pylint: disable=missing-function-docstring
```

Comment:
I think it would be better to move this to the `messages_control.disable` list in pyproject.toml.

I often see people writing completely redundant (useless) docstrings just to appease the linter. In my experience it is better left for the review stage, requesting docstrings as needed, if something isn't clear. (Even then, I usually encourage clearer variable and function naming over adding docstrings.)

Reply:
We already have docstrings on everything except `__main__` (where they are very much not needed), and I don't expect this to change much. I did disable the module docstring one, as that would be overkill. How about we only disable it (further) if it gets in the way?

Reply:
It's fine.
Internally we have our linter configured to warn about function docstrings only if the function is more than N lines (10 maybe, not sure). I find that useful.
But we can revisit as needed.